docs/2-tutorials/2-develop/6-push-notifications.html.md

```diff
@@ -2,7 +2,164 @@
+
+6. Type `$ phonegap serve`. You will receive the server address the app is being hosted on in the output received in the console (`192.168.1.11:3000` in this example):
+
+		$ phonegap serve
+		[phonegap] starting app server...
+		[phonegap] listening on 192.168.1.11:3000
+		[phonegap]
+		[phonegap] ctrl-c to stop the server
+		[phonegap]
+
+7. Now go to your mobile device where the PhoneGap Developer App is running, enter the server address on the main screen and tap **Connect**.
+
+  <img class="mobile-image" src="/images/dev-app-enter-add.png" alt="PhoneGap Developer App, iOS"/>
+
+
+ <div class="alert--info"> **NOTE:** Tap directly on the server address displayed in the terminal screen of the PhoneGap Developer app to change it to match yours. The value filled in by default is only a sample. </div>
+
+ You should see the connection occur followed by a success message as shown below. If you receive an error of any kind, ensure
+ once again that you are connected to the same network on both your  and computer. You could also check the
+ [issue tracker](https://github.com/phonegap/phonegap-app-developer/issues) and [PhoneGap Google Groups](https://groups.google.com/forum/#!forum/phonegap)
+ list for further help.
+
+  <img class="mobile-image" src="/images/dev-app-success.jpg" alt="Developer App, connection success"/>
+
+
+ Once the PhoneGap Developer app connects and loads your mobile application, it should be displayed for preview as shown below:
+
+ <img class="mobile-image" src="/images/dev-app-preview.jpg" alt="Developer App, preview"/>
+
+ The grey `Requesting Registration ID` line under the PhongGap Bot should turn into a green `Registered` line and you should see the following in your terminal session.
+
+     [phonegap] [console.log] registration event: APA91bE1MmeTc92igNoi5OkDWUV
+
+ <div class="alert--warning">Obviously, your registration ID will be different but you get the picture.</div>
+
+### Sending Your First Push
+
+1. Open a new terminal window and enter the following command if you are testing on iOS:
+
+		$ phonegap push --deviceID APA91bE1MmeTc92igNoi5OkDWUV --service apns --payload '{ "aps": { "alert": "Hello World" } }'
+
+   or enter the following command if you are testing on Android:
+
+		$ phonegap push --deviceID APA91bE1MmeTc92igNoi5OkDWUV --service gcm --payload '{ "data": { "title": "Hello", "message": "World"} }'
+
+   replacing the deviceID with the one you received from the registration event in the previous step.
+
+2. This will result in a push notification being received by the app.
+
+   <img class="mobile-image" src="/images/dev-app-success.jpg" alt="Developer App, connection success"/>
+
+### Explaining Push
+
+Now open the **www/index.js** file and we'll take a deep dive into what is going on. All of the code required to register and receive push notifications is in the `onDeviceReady` function.
```

Comment:
Update this to the right path: **www/index.js** should be **www/js/index.js**

And I would change this sentence:
>All of the code required to register and receive push notifications is in the `onDeviceReady` function.

to

> All of the code required to register and receive push notifications is in the `setupPush` function (called from the `onDeviceReady` function).

Or something to that effect to avoid confusion...